tests/VirtoCommerce.ElasticAppSearch.Tests/SearchQueryBuilderTests.cs

```diff
@@ -86,6 +87,82 @@ public void ToSearchQuery_ConvertValidSearchFields_Successfully(string[] searchF
             Test(() => new SearchRequest { SearchFields = searchFields }, searchQuery => searchQuery.SearchFields, expectedResult);
         }
 
+        [Theory]
+        [InlineData(new object[] { new[] { "test" }, "{\"test\":{}}" })]
+        [InlineData(new object[] { new[] { "test1", "test2" }, "{\"test1\":{},\"test2\":{}}" })]
+        public void TestSearchFields(string[] searchField, string result)
+        {
+            // Arrange
+            var searchQueryBuilder = new SearchQueryBuilder(GetFieldNameConverter(), GetSearchFiltersBuilder());
+
+            var request = new SearchRequest
+            {
+                SearchFields = new List<string>(searchField),
+            };
+
+            // Act
+            var searchQuery = searchQueryBuilder.ToSearchQuery(request, GetSchema());
+            var serializedQuerySearchFields = JsonConvert.SerializeObject(searchQuery.SearchFields);
+
+            // Assert
+            Assert.Equal(result, serializedQuerySearchFields);
+        }
+
+        [Fact]
+        public void NullSearchFields()
+        {
+            // Arrange
+            var searchQueryBuilder = new SearchQueryBuilder(GetFieldNameConverter(), GetSearchFiltersBuilder());
+
+            var request = new SearchRequest { SearchFields = null, };
+
+            // Act
+
+            var result = searchQueryBuilder.ToSearchQuery(request, GetSchema());
+
+            // Assert
+
+            Assert.NotNull(result);
+            Assert.Null(result.SearchFields);
+        }

```

Comment:
These tests was refactored and splitted to serialization tests (`SearchFieldsSerializationTest`) and conversion (VC -> API) tests (`ToSearchQuery_ConvertValidSearchFields_Successfully` above). So you may simply remove them